src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -4,15 +4,15 @@
 	font-style: normal;
 	font-weight: 400;
 	src: local('Fira Sans'),
-		url("FiraSans-Regular.woff2") format("woff2");
+		url("FiraSans-Regular-018c141bf0843ffd.woff2") format("woff2");
```

Comment:
Does it mean we will have to manually update this hash every time we update these files?

Reply:
Yes. Fortunately, the only files referenced in this way in rustdoc.css almost never change: the fonts and the toggles. As a bonus, we get rid of the awkward AUTOREPLACE functionality.

Reply:
I'm not sure to be a big fan of this considering it means more manual code to handle. The worst part being that the file will very likely exist in the destination folder already, so if we forget to update this entry, it'll still pick the old one. Remembering to change this will be a nightmare...

Reply:
The previous revision of a changed file would only be present in the destination folder for an unclean build (e.g. when developing locally). In CI it would not be present. So, how can we make sure we detect such cases in CI? Is it possible to change browser-ui-test so it treats any failure to load a subresource as failing the test?

Reply:
Currently it only detects JS errors. I think we can make it detect load errors too.